deploy/03_resolve_uniswap_v2_router.ts

```diff
@@ -24,6 +24,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       contract: "UniswapV2RouterStub",
       from: deployer,
       log: true,
+      waitConfirmations: 1,
```

Comment:
Out of curiosity - why don't we add the same for deployments in the earlier scripts?

Reply:
We use stubs on the hardhat network for unit tests and it works stable there.